Conch/t/integration/04_test_datacenter_loaded.t

```diff
@@ -21,7 +20,8 @@ my $dbh = DBI->connect( $pgtmp->dsn );
 my $t = Test::Mojo->new(
 	Conch => {
 		pg      => $pgtmp->uri,
-		secrets => ["********"]
+		secrets => ["********"],
+		features => { rollbar => 0 },
```

Comment:
I disabled this because the exception handler in the rollbar plugin was getting called and hanging the tests.  This is what led to PR #218 so I could make sure the rollbar code wasn't going to get called at all in the tests.